dev/benchmarks/macrobenchmarks/lib/src/web/recorder.dart

```diff
@@ -420,6 +420,7 @@ class Profile {
   Map<String, dynamic> toJson() {
     return <String, dynamic>{
       'name': name,
+      'scoreKeys': <String>['averageDrawFrameDuration'],
```

Comment:
For consistency with the devicelab nomenclature let's call this "benchmarkScoreKeys"

Reply:
I have another, larger PR sitting on top of this PR (I'll send it your way soon). Would you mind if I make this change in the other PR? It's easier that way.